moviesite/polls/movie_api.py

```diff
@@ -0,0 +1,49 @@
+import re
+import json
```

Comment:
Its a good practice. Keeps things better organized. You won't find it in pep8 but we strictly follow it.